cmd/argocd/commands/repo.go

```diff
@@ -113,9 +113,9 @@ func NewRepoListCommand(clientOpts *argocdclient.ClientOptions) *cobra.Command {
 			repos, err := repoIf.List(context.Background(), &repository.RepoQuery{})
 			errors.CheckError(err)
 			w := tabwriter.NewWriter(os.Stdout, 0, 0, 2, ' ', 0)
-			fmt.Fprintf(w, "REPO\tUSER\n")
+			fmt.Fprintf(w, "REPO\tUSER\tERROR\n")
 			for _, r := range repos.Items {
-				fmt.Fprintf(w, "%s\t%s\n", r.Repo, r.Username)
+				fmt.Fprintf(w, "%s\t%s\n", r.Repo, r.Username, r.Error)
```

Comment:
One `%s` is missing. Linter is unhappy: http://argo.applatix.net/workflows/default/argo-cd-ci-s4dlc?tab=workflow&nodeId=argo-cd-ci-s4dlc-4209827624&sidePanel=logs%3Aargo-cd-ci-s4dlc-4209827624%3Amain